unlock-app/src/components/content/LogContent.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import Layout from '../interface/Layout'
+import Account from '../interface/Account'
+import BrowserOnly from '../helpers/BrowserOnly'
+import GlobalErrorConsumer from '../interface/GlobalErrorConsumer'
+import CreatorLog from '../creator/CreatorLog'
+import { pageTitle } from '../../constants'
+import withConfig from '../../utils/withConfig'
+import * as UnlockTypes from '../../unlock'
+
+interface Props {
+  account: UnlockTypes.Account
+  network: UnlockTypes.Network
+  transactionFeed: UnlockTypes.Transaction[]
+  explorerLinks: { [key: string]: string }
+}
+
+export const LogContent = ({ account, network, transactionFeed, explorerLinks }: Props) => {
+  return (
+    <GlobalErrorConsumer>
+      <Layout title="Creator Log">
+        <Head>
+          <title>{pageTitle('Log')}</title>
+        </Head>
+        {account && (
+          <BrowserOnly>
+            <Account network={network} account={account} />
+            <CreatorLog transactionFeed={transactionFeed} explorerLinks={explorerLinks} />
+          </BrowserOnly>
+        )}
+      </Layout>
+    </GlobalErrorConsumer>
+  )
+}
+
+interface State {
+  account: UnlockTypes.Account
+  network: UnlockTypes.Network
+  transactions: UnlockTypes.Transactions
+}
+
+interface Config {
+  chainExplorerUrlBuilders: UnlockTypes.ChainExplorerURLBuilders
+}
+
+export const mapStateToProps = (
+  { account, network, transactions }: State,
+  { config: { chainExplorerUrlBuilders } }: { config: Config }
+) => {
```

Comment:
can we specify a return type as well here? Or does it mess with redux typings?

Reply:
Adding a return type here works, I've pushed a change with that in there (it's just `: Props` before the `=>`)